src/Umbraco.Web.BackOffice/Extensions/IdentityBuilderExtensions.cs

```diff
@@ -24,6 +25,19 @@ public static class IdentityBuilderExtensions
             return identityBuilder;
         }
 
+        /// <summary>
+        /// Adds a <see cref="UserManager{TUser}"/> for the <seealso cref="MembersIdentityUser"/>.
+        /// </summary>
+        /// <typeparam name="TInterface">The usermanager interface</typeparam>
+        /// <typeparam name="TUserManager">The usermanager type</typeparam>
+        /// <returns>The current <see cref="IdentityBuilder"/> instance.</returns>
+        public static IdentityBuilder AddMembersUserManager<TInterface, TUserManager>(this IdentityBuilder identityBuilder)
```

Comment:
Similar to above, these need to be part of Common since they will be used in the front-end too. The Front end assembly does not reference the back office. They can be shipped separately.

Reply:
Good point - have moved